doc/source/changes/version_0_26.rst.inc

```diff
@@ -44,6 +44,27 @@ Miscellaneous improvements
 * replaced `na` argument of `read_csv`, `read_excel`, `read_hdf` and `read_sas` functions by `fill_value`
   (closes :issue:`394`)
 
+* allowed to pass a label or a named group as `sheet_name` argument of the method `to_excel` and
+  `key` argument of the method `to_hdf` (closes :issue:`328`):
+
+    >>> arr = ndtest((4, 4, 4))
+
+    >>> # iterate over labels of a given axis
+    >>> for label in arr.a:
+    ...     arr[label].to_excel('my_file.xlsx', label)
+    ...     arr[label].to_hdf('my_file.h5', label)
+
+    >>> # create and use a group
+    >>> even = arr.a['a0::2'] >> 'even'
```

Comment:
I don't think using ::2 in an example is worth it. It is likely to confuse most users because they do not know that syntax.

```python
mygroup = arr.a['a0,a2,a3'] >> 'mygroup'
```

Reply:
They don't know that syntax? I guess you're right. I'll change the example as you suggest.